nipype/interfaces/base/core.py

```diff
@@ -587,6 +587,11 @@ def run(self, cwd=None, ignore_exception=None, **inputs):
 
         return results
 
+    def __call__(self, **kwargs):
+        """Make interface into a callable function"""
+        results = self.run(**kwargs)
+        return results.outputs
```

Comment:
what about this to match with the call to `run`?

```suggestion
        return results
```

Reply:
I have zero expectations of behavior here, so sure?

Reply:
Dask might expect something like `results.outputs`.